### PR TITLE
add media for ipad mini

### PR DESCRIPTION
**Pull request by glanttz**

### CODE REVIEW

**Comment by glanttz** [comment]
i sat with this today and repaired. now every range has own style (i add min-width and range from min to max)

File: src/components/ButtonComponent.vue [hunk 1]
```diff
@@ -181,4 +181,25 @@
             color: $blue-700;
         }
     }
+    @media (max-width: $tablet-width) {
```
Comment (PixelSculptor) [inline src/components/ButtonComponent.vue:190]: good implementation in reusable component

File: src/styles/model/variables.scss [hunk 2]
```diff
@@ -75,3 +75,34 @@ $button-width-wide: 22rem;
...
+
+//max-width for media
+$zero-width: 0px;
+$tablet-width: 768px;
+$tablet-above-width: 769px;
+$laptop-width: 1024px;
+$laptop-above-width: 1025px;
+$pc-width: 1370px;
+
+//tablets
```
Comment (PixelSculptor) [inline src/styles/model/variables.scss]: this numbers are from experience or youve searched for appropriate mobile design on Internet?

File: src/components/prescriptions/PrescriptionComponent.vue [hunk 3]
```diff
@@ -119,4 +119,79 @@
             @include text-header6($font-weight-regular);
         }
     }
+
+    @media (min-width: $zero-width) and (max-width: $tablet-width) {
```
Comment (PixelSculptor) [inline src/components/prescriptions/PrescriptionComponent.vue]: Not sure about min width with 0 is necessary? If range is 0 < x<= e.g. 750px its not required to declare min-with 0px where containers does not negative width

File: src/components/prescriptions/PrescriptionComponent.vue [hunk 4]
```diff
@@ -119,4 +119,79 @@
...
+                text-align: center;
+            }
+            &__data {
+                @include text-header6--tablets($font-weight-medium);
+                text-align: center;
+            }
+        }
+    }
+
+    @media (min-width: $tablet-above-width) and (max-width: $laptop-width) {
```
Comment (PixelSculptor) [inline src/components/prescriptions/PrescriptionComponent.vue]: why var above it? I think the better idea is to set min-width for laptop width and delete max width . Then next grade of media will be starting from min width e.g 1024px

File: src/components/prescriptions/PrescriptionComponent.vue [hunk 5]
```diff
@@ -119,4 +119,79 @@
...
+                text-align: center;
+            }
+            &__data {
+                @include text-header5--laptops($font-weight-medium);
+                text-align: center;
+            }
+        }
+    }
+
+    @media (min-width: $laptop-above-width) and (max-width: $pc-width) {
```
Comment (PixelSculptor) [inline src/components/prescriptions/PrescriptionComponent.vue]: same here. maybe declare only low border of range without closing

File: src/components/prescriptions/PrescriptionImportComponent.vue [hunk 6]
```diff
@@ -0,0 +1,163 @@
+<template>
```
Comment (PixelSculptor) [inline src/components/prescriptions/PrescriptionImportComponent.vue:1]: is it new component? there are tons of lines which has marked as new?

File: src/components/referrals/ImportReferralsLabels.vue [hunk 7]
```diff
@@ -76,4 +76,80 @@
...
+                text-align: center;
+            }
+            &__data {
+                @include text-header5--laptops($font-weight-medium);
+                text-align: center;
+            }
+        }
+    }
+
+    @media (min-width: $laptop-above-width) and (max-width: $pc-width) {
```
Comment (PixelSculptor) [inline src/components/referrals/ImportReferralsLabels.vue]: range

File: src/components/referrals/ReferralsComponentLabel.vue [hunk 8]
```diff
@@ -119,4 +119,79 @@
             height: 3rem;
         }
     }
+
+    @media (min-width: $zero-width) and (max-width: $tablet-width) {
```
Comment (PixelSculptor) [inline src/components/referrals/ReferralsComponentLabel.vue]: as above I dont think is it good solution

File: src/components/referrals/ReferralsGroupComponent.vue [hunk 9]
```diff
@@ -72,5 +72,94 @@
...
+                gap: 1rem;
+                margin-left: 5rem;
+            }
+        }
+
+        @media (min-width: $tablet-above-width) and (max-width: $laptop-width) {
+            &__header {
+                @include text-header3($font-weight-semiBold);
+                color: $blue-900;
+                @include flex-position(row, nowrap, flex-start, center);
```
Comment (PixelSculptor) [inline src/components/referrals/ReferralsGroupComponent.vue:112]: include up

File: src/components/referrals/ReferralsGroupComponent.vue [hunk 10]
```diff
@@ -72,5 +72,94 @@
...
+                gap: 1rem;
+                .numOfRefs {
+                    display: inline-block;
+                    height: 2.5rem;
+                    width: 2.5rem;
+                    @include flex-position(column, nowrap, center, center);
```
Comment (PixelSculptor) [inline src/components/referrals/ReferralsGroupComponent.vue:118]: upper

File: src/components/referrals/ReferralsGroupComponent.vue [hunk 11]
```diff
@@ -72,5 +72,94 @@
...
```
Comment (PixelSculptor) [inline src/components/referrals/ReferralsGroupComponent.vue]: upper

File: src/view/PrescriptionView.vue [hunk 12]
```diff
@@ -137,5 +137,75 @@
             border-top-left-radius: $border-radius--rounded;
             border-bottom-left-radius: $border-radius--rounded;
         }
+        @media (min-width: $zero-width) and (max-width: $tablet-width) {
```
Comment (PixelSculptor) [inline src/view/PrescriptionView.vue]: dont think is it good with 0 px

File: src/view/PrescriptionView.vue [hunk 13]
```diff
@@ -137,5 +137,75 @@
...
+                justify-content: flex-start;
+                margin-left: 5rem;
+            }
+            &__actions {
+                grid-area: actions;
+                place-self: space-between;
+                width: 70%;
+                gap: 2rem;
+                margin-left: 5rem;
+                @include flex-position(row, nowrap, flex-start, center);
```
Comment (PixelSculptor) [inline src/view/PrescriptionView.vue:159]: up

File: src/view/PrescriptionView.vue [hunk 14]
```diff
@@ -137,5 +137,75 @@
             border-top-left-radius: $border-radius--rounded;
             border-bottom-left-radius: $border-radius--rounded;
         }
+        @media (min-width: $zero-width) and (max-width: $tablet-width) {
+            &__header {
+                height: 50%;
+                width: 70%;
+                grid-area: dashboard;
+                @include text-header1($font-weight-semiBold);
```
Comment (PixelSculptor) [inline src/view/PrescriptionView.vue:148]: up

File: src/view/ReferralsView.vue [hunk 15]
```diff
@@ -139,5 +139,76 @@
             border-bottom-left-radius: $border-radius--rounded;
         }
         //TODO: Kacper will make changes in component UserInfo so he need to in the future change styles here.
+
+        @media (min-width: $zero-width) and (max-width: $tablet-width) {
+            &__header {
+                height: 50%;
+                width: 70%;
+                grid-area: dashboard;
+                @include text-header1($font-weight-semiBold);
```
Comment (PixelSculptor) [inline src/view/ReferralsView.vue:148]: up

File: src/view/ReferralsView.vue [hunk 16]
```diff
@@ -139,5 +139,76 @@
             border-bottom-left-radius: $border-radius--rounded;
         }
         //TODO: Kacper will make changes in component UserInfo so he need to in the future change styles here.
+
+        @media (min-width: $zero-width) and (max-width: $tablet-width) {
```
Comment (PixelSculptor) [inline src/view/ReferralsView.vue]: i am not sure about this

File: src/view/ReferralsView.vue [hunk 17]
```diff
@@ -139,5 +139,76 @@
...
+                display: none;
+            }
+        }
+
+        @media (min-width: $tablet-above-width) and (max-width: $laptop-width) {
+            &__header {
+                height: 50%;
+                width: 70%;
+                grid-area: dashboard;
+                @include text-header1($font-weight-semiBold);
```
Comment (PixelSculptor) [inline src/view/ReferralsView.vue:171]: up

File: src/view/ReferralsView.vue [hunk 18]
```diff
@@ -139,5 +139,76 @@
...
+                justify-content: flex-start;
+                margin-left: 5rem;
+            }
+            &__actions {
+                grid-area: actions;
+                place-self: space-between;
+                width: 70%;
+                gap: 2rem;
+                margin-left: 5rem;
+                @include flex-position(row, nowrap, flex-start, center);
```
Comment (PixelSculptor) [inline src/view/ReferralsView.vue:159]: up

File: src/view/ReferralsView.vue [hunk 19]
```diff
@@ -139,5 +139,76 @@
...
+                display: flex;
+                align-content: center;
+                justify-content: flex-start;
+            }
+            &__actions {
+                grid-area: actions;
+                place-self: space-between;
+                width: 70%;
+                gap: 2rem;
+                @include flex-position(row, nowrap, flex-start, center);
```
Comment (PixelSculptor) [inline src/view/ReferralsView.vue:182]: up

File: src/view/ReferralsView.vue [hunk 20]
```diff
@@ -139,5 +139,76 @@
...
+                display: none;
+            }
+        }
+
+        @media (min-width: $laptop-above-width) and (max-width: $pc-width) {
+            &__header {
+                height: 50%;
+                width: 70%;
+                grid-area: dashboard;
+                @include text-header1($font-weight-semiBold);
```
Comment (PixelSculptor) [inline src/view/ReferralsView.vue:194]: up

File: src/view/ReferralsView.vue [hunk 21]
```diff
@@ -139,5 +139,76 @@
...
+                display: flex;
+                align-content: center;
+                justify-content: flex-start;
+            }
+            &__actions {
+                grid-area: actions;
+                place-self: space-between;
+                width: 70%;
+                gap: 2rem;
+                @include flex-position(row, nowrap, flex-start, center);
```
Comment (PixelSculptor) [inline src/view/ReferralsView.vue:207]: up

**Review by PixelSculptor — COMMENTED** [review]
I am not sure about some of this solutions. Give me time till Saturday when I can focus on this and I will write is it the only solution or we can write it much easier. Fix some changes and see to this 0 < x < Z range issue cause i am not sure is it necessary. Consider some easier solutions. I will investigate this problem with wrapping date as I told. Btw this PR is too big for me - its hard to focus on small changes. There is no very strict deadline and you ought to divide this media fixes into views and components - one view/component - one PR. I am not sure that Ive checked whole issues so its quite dangerous. Then all views are in 1 PR and its not good for me and for u - remember about this next time. I will do my media issues for few PRs so please be ready on Saturday to checking and approving/requesting changes in my PRs.

File: src/styles/model/variables.scss (hunk 2 above)
Comment (glanttz) [reply src/styles/model/variables.scss]: znalazlam takie dane w internecie, gdzie podawali ze takie i takie tu, takie i takie do tego.

File: src/components/prescriptions/PrescriptionComponent.vue (hunk 3 above)
Comment (glanttz) [reply src/components/prescriptions/PrescriptionComponent.vue]: wiesz co, ja to wykonalam ze wzgledu na problem o ktorym pisałam - dla 1024 style dzialaly, dla 1370 działaly a jak doszlo do ipada pobieralo style zamiast do 768 to do 1370 - na stackoverflow znalazlam takie rozwiazanie i zadziałało i dobrze czyta style

File: src/components/prescriptions/PrescriptionComponent.vue (hunk 4 above)
Comment (glanttz) [reply src/components/prescriptions/PrescriptionComponent.vue]: zrobilam to by zgodnosc byla wszedzie, oznacza to ze jak dodalam z tym zerem to chcialam to wszedzie dodac by bylo jednolicie.

File: src/components/prescriptions/PrescriptionComponent.vue (hunk 5 above)
Comment (glanttz) [reply src/components/prescriptions/PrescriptionComponent.vue]: jak wyzej

File: src/components/referrals/ImportReferralsLabels.vue (hunk 7 above)
Comment (glanttz) [reply src/components/referrals/ImportReferralsLabels.vue]: jak wyzej

File: src/components/prescriptions/PrescriptionImportComponent.vue (hunk 6 above)
Comment (glanttz) [reply src/components/prescriptions/PrescriptionImportComponent.vue:1]: tu byl problem ze zauwazylam ze plik sie zle nazywal, byl "prescriptionImportComponent..vue" , z dwoma kropkami. porpawilam blad po prostu

File: src/components/referrals/ReferralsComponentLabel.vue (hunk 8 above)
Comment (glanttz) [reply src/components/referrals/ReferralsComponentLabel.vue]: jak wyzej

File: src/view/PrescriptionView.vue (hunk 12 above)
Comment (glanttz) [reply src/view/PrescriptionView.vue]: jak wyzej

File: src/view/ReferralsView.vue (hunk 16 above)
Comment (glanttz) [reply src/view/ReferralsView.vue]: jak wyzej

**Review by PixelSculptor — APPROVED** [review]

**Pull request merged**